ome/omero-guide-cellprofiler
# Cp4

This PR upgrades CellProfiler to version 4 which has Python 3 support
Update so we can engage on zarr
cc @pwalczysko

To test:
* Check that the notebooks still work
* Check the scripts

Edit: Bump to 4.1.3 where the problem described below has been fixed
~~There was a problem with the version of numpy used by cp
see https://github.com/CellProfiler/CellProfiler/issues/4353~~
